components/coachmark/index.css

```diff
@@ -120,7 +119,11 @@ governing permissions and limitations under the License.
 .spectrum-CoachMark-action-menu {
 	white-space: nowrap;
 	z-index: 1;
-	margin-inline-start: var(--mod-coachmark-heading-to-action-button, var(--spectrum-coachmark-heading-to-action-button))
+	margin-inline-start: var(--mod-coachmark-heading-to-action-button, var(--spectrum-coachmark-heading-to-action-button));
+
+	& .spectrum-Popover {
+		--mod-popover-filter: drop-shadow(var(--mod-popover-drop-shadow-x, var(--spectrum-popover-drop-shadow-x)) var(--mod-popover-drop-shadow-y, var(--spectrum-popover-drop-shadow-y)) var(--mod-popover-drop-shadow-blur, var(--spectrum-popover-drop-shadow-blur)) var(--mod-popover-drop-shadow-color, var(--spectrum-popover-drop-shadow-color)));
+	}
```

Comment:
I think this needs some help. For the record, none of these variables are defined. I just copied them verbatim from the popover CSS. I was trying to add the drop shadow back onto this popover, since it's lying on top of the rest of the coach mark's popover. I assume that's what is supposed to happen given this note: "Containers that appear on top of content, such as menus and tooltips, have drop shadows to show elevation" in Figma, but then also the CSS comment to "prevent nested popovers from affecting each others drop shadow filters" (found here)

Reply:
Doing this is also introducing extra mods to coach mark and I'm not sure we _need_ to do that.

Reply:
As far as I can tell after some research, the `filter(drop-shadow)` being used will be affected by the children and there isn't a way to exclude the nested popover from affecting the coachmark popover edge.

We use `drop-shadow` instead of `box-shadow` on popover in order for the shadow to also follow the triangle tip. So one thought I had is that in popover we could use `box-shadow` instead only when there is no tip displayed, like for coachmark. I think that would solve this situation, since coachmark doesn't appear to have a tip option. I'd like to get others thoughts on that idea.

If there are other situations where the popover _with a tip_ has an issue with these nested shadows, the only other workaround I saw was applying the filter only to a separate element like a pseudo element---that doesn't sound great, since it'd be a messy markup change, likely involving an empty element and moving where the background and tip psuedo element live.